.changeset/eighty-bobcats-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/vercel': major
```

Comment:
Changing the type of version bump we want is easy luckily. You can update the file here with `major`, `minor` or `patch`. This should probably be a `minor` as it adds a new feature.

```suggestion
'@astrojs/vercel': minor
```